tests/foreman/api/test_gpgkey.py

```diff
@@ -28,3 +41,213 @@ def test_positive_search_in_org(self):
         gpg_keys = gpg_key.search({'organization'})
         self.assertEqual(len(gpg_keys), 1)
         self.assertEqual(gpg_key.id, gpg_keys[0].id)
+
+    @tier1
+    @run_only_on('sat')
+    def test_positive_create_with_name(self):
+        """Create a GPG key with valid name.
+
+        @Assert: A GPG key is created with the given name.
+
+        @Feature: GPGKey
+        """
+        for name in valid_data_list():
+            with self.subTest(name):
+                gpg_key = entities.GPGKey(
+                    organization=self.org, name=name).create()
+                self.assertEqual(name, gpg_key.name)
+
+    @tier1
+    @run_only_on('sat')
+    def test_positive_create_with_content_text(self):
+        """Create a GPG key with valid name and valid gpg key text.
+
+        @Assert: A GPG key is created with the expected content.
+
+        @Feature: GPGKey
+        """
+        gpg_key = entities.GPGKey(
+            organization=self.org, content=self.key_content).create()
+        self.assertEqual(self.key_content, gpg_key.content)
+
+    @tier1
+    @run_only_on('sat')
+    def test_positive_create_with_content_path(self):
+        """Create a GPG key with valid name and valid gpg key location.
+
+        @Assert: A GPG key is created with the expected content.
+
+        @Feature: GPGKey
+        """
+        gpg_key = entities.GPGKey(
+            organization=self.org, content=self.key_path).create()
+        self.assertEqual(self.key_path, gpg_key.content)
```

Comment:
Could you please explain what's going on in this test? It looks like you're setting key value to string, containing path to a file? For what reason? `content` is a string field and obviously will store either key or 'path' or whatever you'd like to set. I just can't get why is it needed, doc says following:
```
content
required 	

public key block in DER encoding

Value: String 
```
so the value should be a key, not path to a file. Am i missing something?